### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by vanshitamoolchandani**
Created Home page

### CODE REVIEW

**Review by prankurpandeyy — APPROVED** [review]

**Comment by prankurpandeyy** [comment]
looks good , just give some margin between card content ,rest is good.

File: root.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+@import url('https://fonts.googleapis.com/css2?family=Comfortaa:wght@500;600;700&family=Yellowtail&display=swap');
+:root {
+    --primary-color: #43AA8B  !important;
```
Comment (kgajriaAgarwal) [inline root.css]: do not use !important

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+    background-color: var(--primary-vlight-color);
+}
+
+.img-ctg{
+    height: 12rem;
+}
+
+
+.ctg-layout .btn-link{
+    left: 71px;
```
Comment (kgajriaAgarwal) [inline styles.css]: use rems!

**Review by kgajriaAgarwal — CHANGES_REQUESTED** [review]
image
This space is not looking good.rest everything's good

File: root.css (hunk 1 above)
Comment (vanshitamoolchandani) [reply root.css]: done

File: index.html [hunk 3]
```diff
@@ -0,0 +1,271 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="/styles.css" rel="styleSheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://kit.fontawesome.com/c8ee3dd930.js"></script>
```
Comment (Sahil25061999) [inline index.html]: write the stylesheet link of the homepage after the component link
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 <link href="/styles.css" rel="styleSheet">
 
This way you wont have to use !important.
Also place the script tag at the end in the body

**Review by Sahil25061999 — APPROVED** [review]
Give alt attributes to the image tag

File: index.html (hunk 3 above)
Comment (vanshitamoolchandani) [reply index.html]: done

**Pull request merged**